airbyte-integrations/connectors/source-postgres/build.gradle

```diff
@@ -14,7 +14,7 @@ java {
 airbyteJavaConnector {
     cdkVersionRequired = '0.23.19'
     features = ['db-sources', 'datastore-postgres']
-    useLocalCdk = false
+    useLocalCdk = true
```

Comment:
for running ci on GH only. If passed will need to release cdk and a new version of postgres